source/_components/azuredns.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "Azure DNS"
+description: "Keep your Azure DNS IP address in sync with your external IP address"
+date: 2019-02-23 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: azure.png
+ha_category: Network
+featured: false
+ha_release: 0.90
+---
+
+With the `azuredns` component, users can keep their external IPv4 address in sync with an Azure DNS record. To use this component, you need:
+- An Azure account & subscription
+- An Azure AD App Registration, [which can be configured in the Azure Portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#create-an-azure-active-directory-application)
+- An Azure DNS Zone with an A-record, [which can be configured in the Azure Portal as well](https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal)
+
+You also need to assign the DNS Zone Contributor role to the Azure AD App. You can do this within the DNS-record on the Azure AD Portal.
+
+## {% linkable_title Configuration %}
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+azuredns:
+  domain: example.com
+  host: hass
```

Comment:
If `host` is not required you can remove it

Reply:
Thanks, good one. Fixed in 8c8d1084e0d08ed0332df7c593a65607a771f243.